trip/views.py

```diff
@@ -15,8 +15,6 @@ def create_trip(request):
         if usertrip_creation_form.is_valid():
             usertrip_data = usertrip_creation_form.cleaned_data
 
-            usertrip_instance = usertrip_creation_form.save(commit=False)
```

Comment:
why are we removing this?

Reply:
Not removing it, using `get_or_create` function to handle the creation part.